### PR TITLE
added layout 60_ansi_split_space_bs to dz60 (for qmk configurator)

### PR DESCRIPTION
**Pull request by danceswithwolves**
Added layout 60_ansi_split_space_bs to dz60 for benefit of QMK Configurator's users

## Description

1. Added `#define LAYOUT_60_ansi_split_space_bs` to `/keyboards/dz60/dz60.h`
2. Added `LAYOUT_60_ansi_split_space_bs` to `/keyboards/dz60/info.json` 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dz60/rules.mk [hunk 1]
```diff
@@ -55,4 +55,4 @@ BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality
 AUDIO_ENABLE = no
 RGBLIGHT_ENABLE = yes
 
-LAYOUTS = 60_ansi 60_iso 60_hhkb
+LAYOUTS = 60_ansi 60_iso 60_hhkb 60_ansi_split_space_bs
```
Comment (noroadsleft) [inline keyboards/dz60/rules.mk]: This change should be reverted. This rule is only for the [Community Layouts] feature. There is no `60_ansi_split_space_bs` folder in `layouts/default/`, so this change is not needed.

```suggestion
LAYOUTS = 60_ansi 60_iso 60_hhkb
```

[Community Layouts]: https://docs.qmk.fm/#/feature_layouts

Comment (drashna) [reply keyboards/dz60/rules.mk]: (at least, not yet?)

**Comment by mechmerlin** [comment]
What exactly is this layout accomplishing that one of the other layouts are not able to?

File: keyboards/dz60/rules.mk (hunk 1 above)
Comment (noroadsleft) [reply keyboards/dz60/rules.mk]: :point_up_2:

**Pull request closed without merging**